### ISSUE TITLE
Using ServerFn in dioxus_desktop

### ISSUE THREAD

**Issue by MassiminoilTrace** [DioxusLabs/docsite#145]
I've tried the dioxus_fullstack example. As far as I understood it has an underlying axum server, which serves both the static files and the generated APIs made by ServerFn.

Is it possible to separate them for use in dioxus_desktop?

I imagine something like:
- dioxus_desktop runs a compiled binary (spawns a webview anyway)
- dioxus (ssr) runs server-side axum, serving only the APIs without the web static files
- they talk over the network

I tried to search in the documentation and examples by I haven't found anything so far

**Comment by ealmloff** [comment]
Yes! You can have an Axum server with a desktop/tui/native/mobile application with server functions. There is an example [here](https://github.com/DioxusLabs/dioxus/tree/master/packages/fullstack/examples/axum-desktop).

It would be great to add a small section in the docs explaining how to use dioxus-fullstack with desktop

**Comment by MassiminoilTrace** [comment]
Thanks

I can try to add the section to the docs, after I get a bit more acquainted

**Comment by ealmloff** [comment]
Fixed in https://github.com/DioxusLabs/docsite/pull/149
